Feature/ms 3987 maxupload

EofChris opened the pull request.

extracts committed:
#3987: when uploading or editing attachments on an item's detail view, the server's maximum upload size is now reported above the upload control

extracts committed:
#3987: reverts c7aa871 since the info about the maximum upload file size limit shall be displayed inside the file widget (instead as a label)

extracts committed:
#3987: when uploading or editing attachments on an item's detail view, the info about the server's maximum upload file size limit is now displayed inside the file widget

extracts committed:
just some code formatting changes

extracts committed:
#3987: in the legacy portal configuration, removes links to the "Data upload" config panel since portal- and room-specific upload file size limits are currently not supported

EofChris commented on config/services.yaml:
```diff
@@ -944,11 +944,16 @@ services:
 
-    # Form type extension
+    # Form type extensions
+    commsy.form.extension.upload_file_type:
+        class: App\Form\Extension\UploadFileTypeExtension
+        tags:
+            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType, priority: 1 }
+
     commsy.form.extension.image_type:
         class: App\Form\Extension\ImageTypeExtension
         tags:
-            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType }
+            - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FileType, priority: 0 }
```
Registration of form type extensions is not needed, this is already done by autoconfiguration, see [https://symfony.com/doc/4.4/form/create_form_type_extension.html#registering-your-form-type-extension-as-a-service](https://symfony.com/doc/4.4/form/create_form_type_extension.html#registering-your-form-type-extension-as-a-service)

extracts replied:
I've added the service configuration explicitly since this would allow to use the `priority` attribute (to specify a load order). But I'll try again without…

extracts committed:
#3987: removes the service configuration of form type extensions

due to autoconfiguration, registration of form type extensions isn't needed anymore
